modules/ROOT/pages/mergetags.adoc

```diff
@@ -0,0 +1,55 @@
+= Merge Tags plugin
+:navtitle: Merge Tags
+:description: Enables Merge Tags functionality in the {productname} editor
+:keywords: merge, tags, tag, mergetags, mergetag, mergetags_list, mergetags_prefix, mergetags_suffix, mail merge, personalization token
+:pluginname: Merge Tags
+:plugincode: mergetags
+:pluginminimumplan: tiertwo
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a merge tag (also known as a personalization token, or a mail merge field) from a drop-down list when a specified prefix is typed.
```

Comment:
```suggestion
The {pluginname} plugin allows the user to insert a merge tag (also known as a personalization token, or a mail merge field), either as-you-type, from a drop-down list when a specified prefix is typed, or from a searchable toolbar menu button.
```
Should specify that the toolbar button is the other method of inserting a merge tag and allows searching for a specific tag.

Reply:
I think this should also mention that mergetags are converted from content inserted into the editor or loaded. Maybe it needs to be a list of things some where. Might be a bit heavy for a introduction text. Also that would convert any thing that maches that pattern of `<prefix><text><suffix>`.

- [ ] We need to somehow cover the fact that mergetags will be identified from loaded/inserted content as well

Reply:
Added a slightly modified version of @MitchCTiny’s suggested change.

Also, added a usage note re TinyMCE auto-recognising strings that match an existing <prefix><merge-tag><suffix> when such a string is added to a document.